src/state.rs

```diff
@@ -133,10 +133,9 @@ impl State {
 				if let Some(channel) = state.channel_id {
 					// channel id available, insert voice state
 					if let Some(call) = self.calls.get_mut(&channel) {
-						match call.voice_states.iter_mut().find(|u| u.user_id == state.user_id) {
-							Some(grp_state) => { grp_state.clone_from(state); return }
-							None => {},
-						}
+						if let Some(grp_state) = call.voice_states.iter_mut().find(|u| u.user_id == state.user_id) {
+								grp_state.clone_from(state); return
+							}
```

Comment:
Looks like the indentation got a little messed up here and the next diff block, as well as 3 blocks down.